pyproject.toml

```diff
@@ -254,3 +213,25 @@ markers = [
     "no_mock_api_client: skip mocking api client",
 ]
 norecursedirs = ["hooks", "*.egg", ".eggs", "dist", "build", "docs", ".tox", ".git", "__pycache__"]
+
+[tool.ruff]
+preview = true
+extend-exclude = ["scripts", "hive-udf", "docs", "docker", "assets", ".task", ".mypy_cache", "site"]
+line-length = 120
```

Comment:
Is it possible to keep the line-length the same as before at 100?

Reply:
Looks like there are two options for line length in ruff: `line-length` and `lint.pycodestyle.line-length`. 

Would be good if we can keep the current settings: when formatting, use line-length of 100. Flag line too long at 150.

Reply:
altered